shared/cfg/guest-os/Windows/Win2008/i386.cfg

```diff
@@ -1,5 +1,4 @@
 - i386:
-    maxmem = 4G
```

Comment:
this is the details for win2008 mem limitation 
https://msdn.microsoft.com/en-us/library/windows/desktop/aa366778(v=vs.85).aspx#physical_memory_limits_windows_server_2008_r2